packages/core/src/browser/shell/side-panel-handler.ts

```diff
@@ -187,9 +194,9 @@ export class SidePanelHandler {
         return toolbar;
     }
 
-    protected createSidebarBottomMenu(): SidebarBottomMenuWidget {
-        const bottomMenu = this.sidebarBottomWidgetFactory();
-        bottomMenu.addClass('theia-sidebar-bottom-menu');
+    protected createSidebarMenu(): SidebarMenuWidget {
+        const bottomMenu = this.sidebarWidgetFactory();
+        bottomMenu.addClass('theia-sidebar-menu');
```

Comment:
In addition to the breaking changes Vince has noted, changing this classname is also breaking, as it requires adopters to modify their CSS (as you've modified the internal CSS) if they have any rules targeting these items. I understand the impetus to rename things according to their new behavior, but I'm not sure that it adds enough value here to justify the number of breakages it introduces.

Reply:
> In addition to the breaking changes Vince has noted, changing this classname is also breaking, as it requires adopters to modify their CSS (as you've modified the internal CSS) if they have any rules targeting these items. I understand the impetus to rename things according to their new behavior, but I'm not sure that it adds enough value here to justify the number of breakages it introduces.

can i add this to the breaking changes, or you mean we need to refactor this all commit in order to avoid the breaking changes?

Reply:
At your discretion, but do try to avoid breaking changes unless they're necessary for or contribute substantially to enhancing the functionality delivered. On that basis, I would probably stick with the old classname.

Reply:
@colin-grant-work - will you recommend creating a base shared class and deriving the bottom and top from it to avoid the bottom breakage (although for now the behavior is same and derived class will be empty).?